spec/active_record/sql_analyzer/redacted_logger_spec.rb

```diff
@@ -51,6 +51,19 @@
       end
     end
 
+    context "sql quoted" do
+      let(:event) do
+        {
+          caller: [""],
+          sql: "SELECT * FROM foo WHERE name = 'hello\\'s name'"
+        }
+      end
+
+      it "redacts" do
+        expect(filter_event[:sql]).to eq("SELECT * FROM foo WHERE name = '[REDACTED]'")
```

Comment:
So one concern people had was that this promotes non-standard SQL (`[REDACTED]`). 

But then again, this codebase already has some handling for `[REDACTED]` ...